person.rb

```diff
@@ -1,4 +1,8 @@
+require_relative 'nameable'
+
 class Person
+  include Nameable
```

Comment:
- [ ] The Person class should inherit from `Nameable` Kindly make sure to implement this instead of including `Nameable` in the `Person` class

```suggestion
class Person < Nameable
```